### PR TITLE
ImagePixelExtractorTransform support for images with no alpha channel

### LINKED ISSUE #1820: ImagePixelExtractorTransform only supports images with pixel format Format32bppArgb
We also need support for Format24bppRgb.

### PR DESCRIPTION
**Pull request by yaeldekel**
Fixes #1820 .

### CODE REVIEW

File: src/Microsoft.ML.ImageAnalytics/ImagePixelExtractorTransform.cs [hunk 1]
```diff
@@ -542,28 +544,6 @@ private ValueGetter<VBuffer<TValue>> GetGetterCore<TValue>(IRow input, int iinfo
                         else
                         {
                             int idstMin = 0;
-                            if (ex.Alpha)
```
Comment (Ivanidzo4ka) [inline src/Microsoft.ML.ImageAnalytics/ImagePixelExtractorTransform.cs]: >if (ex.Alpha) [](start = 28, length = 13)

why you delete this part?
is it useless?

File: test/data/images/images.tsv [hunk 2]
```diff
@@ -1,3 +1,4 @@
+tomato.bmp	tomato
```
Comment (sfilipi) [inline test/data/images/images.tsv]: >. [](start = 6, length = 1)

did you intend to add a test, or the test using this file is already defined?

Comment (yaeldekel) [reply test/data/images/images.tsv]: There are existing tests that use this file which cover the new functionality after adding this image.

---
In reply to: [238835568](https://github.com/dotnet/machinelearning/pull/1821#discussion_r238835568) [](ancestors = 238835568)

File: src/Microsoft.ML.ImageAnalytics/ImagePixelExtractorTransform.cs (hunk 1 above)
Comment (yaeldekel) [reply src/Microsoft.ML.ImageAnalytics/ImagePixelExtractorTransform.cs]: More or less. The pixel has an 'A' property, which already contains the value 0xFF if the image doesn't have alpha, so no need to special case.

File: test/Microsoft.ML.Tests/Scenarios/TensorflowTests.cs [hunk 3]
```diff
@@ -55,17 +55,17 @@ public void TensorFlowTransforCifarEndToEndTest()
...
 
             prediction = predictFunction.Predict(new CifarData()
             {
                 ImagePath = GetDataPath("images/hotdog.jpg")
             });
             Assert.Equal(0, prediction.PredictedScores[0], 2);
-            Assert.Equal(1, prediction.PredictedScores[1], 2);
-            Assert.Equal(0, prediction.PredictedScores[2], 2);
+            Assert.Equal(0, prediction.PredictedScores[1], 2);
+            Assert.Equal(1, prediction.PredictedScores[2], 2);
```
Comment (abgoswam) [inline test/Microsoft.ML.Tests/Scenarios/TensorflowTests.cs]: Might be a good idea  to verify the  PredictedScores on for  "tomato.bmp" (also "tomato.jpg")

**Comment by Ivanidzo4ka** [comment]
Shall we merge?

**Pull request merged**